app/src/layouts/common/sidebar/organizationsControl/organizationsControl.jsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Parser from 'html-react-parser';
+import { useIntl, defineMessages } from 'react-intl';
+import classNames from 'classnames/bind';
+import { showModalAction } from 'controllers/modal';
+import { withPopover } from 'componentLibrary/popover';
+import { organizationNameSelector, projectNameSelector } from 'controllers/project';
+import ArrowLeftIcon from '../img/arrow-left-inline.svg';
+import OpenPopoverIcon from '../img/open-popover-inline.svg';
+import { OrganizationsPopover } from './organizationsPopover/organizationsPopover';
+import styles from './organizationsControl.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  organization: {
+    id: 'OrganizationsControl.organization',
+    defaultMessage: 'Organization',
+  },
+});
+
+export const OrganizationsControl = () => {
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+  const organizationName = useSelector(organizationNameSelector);
+  const projectName = useSelector(projectNameSelector);
+  const [isOpenPopover, setIsOpenPopover] = useState(false);
+  const onClick = () => {
+    dispatch(
+      showModalAction({
+        id: 'organizationPopover',
+      }),
```

Comment:
Looks like you use the modal engine without rendering modal at all.
Let's try to avoid such cases as it is complicate understanding what happened here.

I'm not clearly understand why you mix modal and popover approaches.
Can it be handled by adding feature to popover to close it by clicking outside?

Reply:
This is a temporary solution to avoid creating a popover store. I'll try to pass the click outside function there. But I'm not sure that this will cover all acceptance criteria.

Reply:
Done.